infra/taskcluster-hook-pipeline-start.json

```diff
@@ -51,7 +51,9 @@
         "retries": 5,
         "routes": [
             "notify.email.release-mgmt-analysis@mozilla.com.on-failed",
-            "notify.irc-channel.#bugbug.on-failed"
+            "notify.irc-channel.#bugbug.on-failed",
+            "project.relman.services.project.bugbug.production.bugbug_data.latest",
```

Comment:
We can just use project.relman.bugbug instead of project.relman.services.project.bugbug

Also, remove 'production' as we currently we only have one channel